wallet/test/Test/Pos/Util/MnemonicSpec.hs

```diff
@@ -130,3 +135,11 @@ spec = do
         $ Aeson.eitherDecode
         $ Aeson.encode
         $ def @(Mnemonic 12)
+
+    -- | Sample json for testing backward compatibility edge case
+    legacy :: BL.ByteString
+    legacy = "{\"bpToList\":[\"derive pulse priority impose execute balcony guard hidden snap unhappy alcohol next\"]}"
```

Comment:
Ditto the above ^

I am a bit wary about having hardcoded mnemonics, I would like to avoid some curious user using these for his wallet.

Perhaps you could use the generator to generate a valid 12 word mnemonic and then use those in this test?

It's pretty much the same thing I did in my [specs](https://github.com/input-output-hk/cardano-sl/blob/develop/wallet-new/test/unit/Test/Spec/CreateWallet.hs#L57) as well.

Reply:
Agreed, even though, for the test vectors, I'd rather keep them as it is. I hardly think a lot of users / exchanges read the source code and it's even less likely they read tests. 

Another option is simply to extend the list of "forbidden" mnemonics with all the one present in the test vectors.

Reply:
https://github.com/input-output-hk/cardano-sl/blob/develop/wallet/src/Pos/Util/Mnemonic.hs#L213-L224